boot/boot.js

```diff
@@ -2167,7 +2168,7 @@ $tw.boot.boot = function() {
 	// Preload any encrypted tiddlers
 	$tw.boot.decryptEncryptedTiddlers(function() {
 		// Startup
-		$tw.boot.startup();
+		$tw.boot.startup(callback);
```

Comment:
I thought `$tw.boot.startup()` now takes an `options` object not a `callback` function.